replication/binlogsyncer.go

```diff
@@ -733,33 +733,61 @@ func (b *BinlogSyncer) parseEvent(s *BinlogStreamer, data []byte) error {
 		// Some events like FormatDescriptionEvent return 0, ignore.
 		b.nextPos.Pos = e.Header.LogPos
 	}
+
+	getCurrentGtidSet := func() GTIDSet {
```

Comment:
can we move these two functions to outside?

Reply:
we can, however having those functions local emphasizes that they are used/called only in scope of `parseEvent`.

same argument applies to having `advanceCurrentGtidSet` local as well.